src/dune_rules/gen_rules.ml

```diff
@@ -163,6 +163,8 @@ end = struct
     | Install_conf.T i ->
       let+ () = install_stanza_rules ~ctx_dir ~expander i in
       empty_none
+    (* TODO: What does this part of the code do? Why does Documentation.t has
+       not the same kind of install_stanza_rules? *)
```

Comment:
This looks like a leftover bit of code or maybe something that was unclear?

Reply:
Yes, something that was unclear... I left this TODO to be "sure" to remember about sorting this out, so fortunately you noticed it.

I can't really understand what is this part of the code about... Removing it has [a very small effect](https://github.com/ocaml/dune/commit/c1d0ae10c3a32154cbbb1ab93d71559f9e5d655a).

I was surprised not to find anything about documentation stanza in this pattern match, since there is something about install stanzas, and documentation stanzas in particular generate install rules?